packages/wdio-local-runner/src/liteWorker.ts

```diff
@@ -0,0 +1,235 @@
+import path from 'node:path'
+import { EventEmitter } from 'node:events'
+import type { WritableStreamBuffer } from 'stream-buffers'
+import type { Options, Workers } from '@wdio/types'
+
+import logger from '@wdio/logger'
+
+import runnerTransformStream from './transformStream.js'
+import ReplQueue from './replQueue.js'
+import RunnerStream from './stdStream.js'
+import type { IsolatedProcess } from './runLite.js'
+import { run } from './runLite.js'
+
+const log = logger('@wdio/lite-runner')
+const replQueue = new ReplQueue()
+const ACCEPTABLE_BUSY_COMMANDS = ['workerRequest', 'endSession']
+
+const stdOutStream = new RunnerStream()
+const stdErrStream = new RunnerStream()
+stdOutStream.pipe(process.stdout)
+stdErrStream.pipe(process.stderr)
+
+/**
+ * WorkerInstance
+ * responsible for spawning a sub process to run the framework in and handle its
+ * session lifetime.
+ */
+export default class LiteWorkerInstance extends EventEmitter implements Workers.Worker {
```

Comment:
Do you think we can extend from the default worker here? It seems like a lot of logic is the same?

Reply:
I can try :-)